python-backend/uwsgi.ini

```diff
@@ -0,0 +1,31 @@
+[uwsgi]
+# Module settings
+callable = create_app()
+module = app
+
+# Curl this address for statistics
+stats = 127.0.0.1:9191
+
+# Server settings
+http-socket = 0.0.0.0:5000
+http-auto-gzip = true
+post-buffering = 8192
+
+# UWSGI settings
+plugins = router_cache
+master = true
+harakiri = 90
+processes = 4
+threads = 2
```

Comment:
Are these tuned for how our Openshift is configured or were these defaults?

Reply:
They're a combination.
The docs said there is no magic number and just to fine tune and play around.
So i started off with this and as we go on we may have to fine tune and play around.